plaso/lib/definitions.py

```diff
@@ -14,8 +14,13 @@
     COMPRESSION_FORMAT_NONE,
     COMPRESSION_FORMAT_ZLIB])
 
+# Default worker process memory limit of 2 GiB.
 DEFAULT_WORKER_MEMORY_LIMIT = 2048 * 1024 * 1024
 
+# Consider a worker process inactive after 15 minutes of no activity
```

Comment:
Change this to just "of no status updates".

Reply:
done